src/context.rs

```diff
@@ -1557,6 +1557,7 @@ impl BlockContext {
     self.for_each(bo, txsize.block_size(), |block| { block.tx_w = tx_w; block.tx_h = tx_h } );
   }
 
+  //TODO: Don'w need set_tx_type()?
```

Comment:
I don't think we need to store tx_type since there are no block to block dependencies

Reply:
Ok- I though some coding tools we haven't implemented yet would use tx_type of neighboring blocks as contexts to choose CDF.